credentials/src/main/java/me/uport/sdk/credentials/Credentials.kt

```diff
@@ -147,6 +148,42 @@ class Credentials(
     }
 
 
+    /**
+     * Verify and return profile from a
+     * [Selective Disclosure Response JWT](https://github.com/uport-project/specs/blob/develop/messages/shareresp.md).
+     *
+     * @param token **REQUIRED** The JWT response token from a selective disclosure request
+     *
+     * @return a [UportProfile] object
+     */
+    @Suppress("TooGenericExceptionCaught")
+    suspend fun verifyDisclosure(token: String): UportProfile {
+
+        val (_, payload, _) = JWTTools().decode(token)
```

Comment:
the response token should actually be verified, not only decoded

Reply:
Alright